task/09-functions-n-closures-tasks.js

```diff
@@ -84,7 +88,11 @@ function getPolynom() {
  *   memoizer() => the same random number  (next run, returns the previous cached result)
  */
 function memoize(func) {
-    throw new Error('Not implemented');
+    let arrResult = func.apply(this, arguments);
+    //probably strange decision )
+    return () => {
+        return arrResult;
+    }
```

Comment:
No, it's not strange :) But you can write it shorter like:
return () => arrResult;